sqm-scripts/patches/002-OverTheBox-DSCP.patch

```diff
@@ -1,18 +1,25 @@
 diff --git a/src/run-openwrt.sh b/src/run-openwrt.sh
-index 97171a6..2fe7d9a 100644
+index 1cf6403..70f2ff0 100644
 --- a/src/run-openwrt.sh
 +++ b/src/run-openwrt.sh
-@@ -23,9 +23,53 @@ if [ "$ACTION" = "stop" -a -z "$RUN_IFACE" ]; then
+@@ -13,6 +13,7 @@
+ 
+ ACTION="${1:-start}"
+ RUN_IFACE="$2"
++IPT="/usr/sbin/iptables -t mangle -w"
+ 
+ [ -d "${SQM_QDISC_STATE_DIR}" ] || ${SQM_LIB_DIR}/update-available-qdiscs
+ 
+@@ -23,9 +24,52 @@ if [ "$ACTION" = "stop" -a -z "$RUN_IFACE" ]; then
          # $SCRIPT variables saved in there.
          [ -f "$f" ] && ( . $f; IFACE=$IFACE SCRIPT=$SCRIPT SQM_DEBUG=$SQM_DEBUG SQM_DEBUG_LOG=$SQM_DEBUG_LOG OUTPUT_TARGET=$OUTPUT_TARGET ${SQM_LIB_DIR}/stop-sqm )
      done
 +    # Clear DSCP rules
```

Comment:
In this file it seems there is a mix of spaces and tabs for indentation. This can cause trouble (merge conflicts) later when we'll edit the file: even if the line hasn't been modified, it will be labeled as modified.